route.rb

```diff
@@ -0,0 +1,39 @@
+class Route
+  attr_accessor :in_between_stations, :starting_station, :ending_station
+
+
+  def initialize(starting_station, ending_station)
+    @starting_station = starting_station
```

Comment:
The array should contain stations, not only names.

Reply:
Not really sure what is the issue, since I populate the array only with the station objects, not with their names